### DISCUSSION TITLE
Class library is empty

### DISCUSSION THREAD

**Discussion by weiweipxbgit** [quartznet/quartznet#1550] [Q&A] [answered]
1.I use get clone the code.

![1](https://user-images.githubusercontent.com/13359332/159217527-c7609e1e-9b9a-4086-b54b-b62ff7280e23.png)
.2. open the sln with the vs2019, and every class library ended with unloaded.
3. and nothing in the class library
![2](https://user-images.githubusercontent.com/13359332/159217636-5f626d9b-cabb-4256-89cd-797d1083e69a.png)

**Comment by lahma** [comment, marked as answer]
Try running `build.cmd` / `build.sh`  and see if it helps. I'm not sure if we currently already require VS2022 for solution to load, you could try the community version if it helps.

**Comment by weiweipxbgit** [comment]
Open it with the vs2022 and It's works, 
Thank you.
![4](https://user-images.githubusercontent.com/13359332/159234341-81914f70-9e44-44c4-82d4-7ec2955260e5.png)

**Reply by lahma** [reply]
Great that it worked. I've updated the main README to contain correct version requirements. Thanks for informing about this!
